### DISCUSSION TITLE
Support for wildcard prompt description

### DISCUSSION THREAD

**Discussion by andupotorac** [Bing-su/adetailer#144] [Ideas]
Right now if I use a wildcard, I don't know which value was selected for the face, as it doesn't show up below the main image.

This is what is shoes up like:
`ADetailer prompt: "(Marc1500:1), detailed __emotions/emotions_anger_all__ face",`
<img width="1257" alt="Screenshot 2023-06-17 at 12 18 28" src="https://github.com/Bing-su/adetailer/assets/3269738/ca25af88-9f7f-4244-880c-100585694898">

**Comment by Bing-su** [comment]
I wanted to do this too, but it's not possible with the current webui structure.

**Reply by andupotorac** [reply]
I hope it gets to a backlog, as it makes it hard to replicate an image from a prompt, as there's no place where we can take this info again. Wouldn't it be possible to send it to the main prompt somehow that is saving in the same metadata?
